tests/acceptance/new-test.js

```diff
@@ -234,6 +234,24 @@ describe('Acceptance: ember new', function() {
     })
   );
 
+  it(
+    'ember new with git blueprint and ref checks out the blueprint with the correct ref and uses it',
+    co.wrap(function*() {
+      this.timeout(20000); // relies on GH network stuff
+
+      yield ember([
+        'new',
+        'foo',
+        '--skip-npm',
+        '--skip-bower',
+        '--skip-git',
+        '--blueprint=https://github.com/buschtoens/app-blueprint-test.git#named-ref',
```

Comment:
```suggestion
        '--blueprint=https://github.com/ember-cli/app-blueprint-test.git#named-ref',
```

Pending https://github.com/ember-cli/app-blueprint-test/pull/5 to be merged.

Reply:
Merged now...